apps/verify/translations/src/en/fields.json

```diff
@@ -14,15 +14,15 @@
   },
   "brp": {
     "label": "Biometric residence permit (BRP) number",
-    "hint": "This must match what is on your BRP card. For example, RZ1234567"
+    "hint": "This must match what is on your BRP card. For example, ZU1234567"
   },
   "uan": {
     "label": "Unique application number (UAN)",
     "hint": "Your UAN is on the Home Office invitation to complete this form. For example, 1111-2222-3333-4444"
   },
   "date-of-birth": {
     "legend": "Date of birth",
-    "hint": "For example, 27 3 1985"
+    "hint": "This must match what is on your BRP card. For example, 27 3 1985"
```

Comment:
I've used the same date-of-birth field for both the UAN page and the BRP one. That means this would update the UAN hint message too. Can we make it dynamic without making date-of-birth-brp and date-of-birth-uan fields?

Reply:
We need to make that a parameter